tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -788,6 +789,36 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
         this.set('_isReloadDisabled', dashboard.disableReload);
       },
 
+      _renderPluginIFrame(container, selectedDashboard, loadingMechanism) {
```

Comment:
caps: `Iframe`, per the [standard camel case definition][1]

[1]: https://google.github.io/styleguide/javaguide.html#s5.3-camel-case